src/pages/Miruro/main.ts

```diff
@@ -0,0 +1,75 @@
+import { pageInterface } from '../pageInterface';
+
+export const Miruro: pageInterface = {
+  name: 'Miruro',
+  domain: ['https://www.miruro.tv', 'https://www.miruro.online'],
+  languages: ['English'],
+  type: 'anime',
+  isSyncPage(url) {
+    return Boolean(j.$('.player[data-media-player]').length);
+  },
+  isOverviewPage(url) {
+    return false;
+  },
+  getImage() {
+    return $('.player[data-media-player] .vds-poster').attr('src');
+  },
+  sync: {
+    getTitle(url) {
+      return utils.urlPart(url, 5).replace(/-/g, ' ').trim();
+    },
+    getIdentifier(url) {
+      // '139630-1' for https://www.miruro.tv/watch/139630/boku-no-hero-academia-6th-season/1
+      return `${utils.urlPart(url, 4)}-${utils.urlPart(url, 6)}`;
+    },
+    getOverviewUrl(url) {
+      const href = `https://${window.location.hostname}/watch/${utils.urlPart(url, 4)}`;
+      if (typeof href !== 'undefined') {
+        return utils.absoluteLink(href, Miruro.domain);
+      }
+      return '';
+    },
+    getEpisode(url) {
+      const temp = utils.urlPart(url, 6);
+      if (!temp) return NaN;
+      return Number(temp);
+    },
+    nextEpUrl(url) {
```

Comment:
Returns episode that does not exist.
https://www.miruro.tv/watch/104459/yuru-camp-season-2/13

Reply:
I'm using the url to generate next episode link. Should I just ignore this function since this is optional?

Reply:
I think i found a way. I'll add a commit/

Reply:
I have pushed the commit. Now it gets the total number of episode in the anime.. and check if next episode is less than equal to the total episode and then returns the url.